src/workspacer.Native/Native/WindowsManager.cs

```diff
@@ -33,7 +33,10 @@ public class WindowsManager : IWindowsManager
         public event WindowCreateDelegate WindowCreated;
         public event WindowDelegate WindowDestroyed;
         public event WindowUpdateDelegate WindowUpdated;
-        public event WindowFocusDelegate WindowFocused;
+
+        public event WindowFocusDelegate OnWindowFocused;
```

Comment:
This event is still renamed :)

Reply:
Oh woops. I missed that this was a pre-existing one when fixing the "trigger" event naming